setup.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+# Copyright 2015: Mirantis Inc.
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+from setuptools import setup
+from setuptools import find_packages
+from pip.req import parse_requirements
+
+setup(name='CloudFerry',
+      version=open('version.txt').read().strip(),
```

Comment:
I guess better way would be to introduce [setup.cfg](https://docs.python.org/2/distutils/configfile.html) where possible, there's plenty of examples online, e.g. [nova's](https://github.com/openstack/nova/blob/master/setup.cfg).